### DISCUSSION TITLE
Part Inspection for suspicions geometry for sheet good materials

### DISCUSSION THREAD

**Discussion by joradnnik** [lairdubois/lairdubois-opencutlist-sketchup-extension#444] [Ideas]
Hello,

Is there a possibility to inspect and suggest suspicions parts. What i mean by that is that usually for sheet goods materials the parts are supposed to be square in all dimensions. It happened to me a few times when trying to finally adjust the entire kitchen assembly to stretch an edge or two without noticing it which changes the geometry and the dimensions as well(which is the worse thing). There was no way later on to see in the generated part list that the new dimensions was modified in wrong way for that specific part.

What i suggest is a kind of post evaluation of the parts made out of a sheet good material for a suspicious geometry like:  
 - not square in all planes
 - overlapping (penetrating) with another parts (geometry) - i know this might be a practical (desired) case however at least such places of contact might be highlighted to bring the attention to those spots. Also if an overlapping is a wished i expect that there will be edges available to the contact points. It is like trimming two objects. When that trimmed the corresponding edges are created in the place of contact 
 - 
Does it make sense ?

Regards
Jordan

**Comment by bbeaulant** [comment]
Hi,

It make sense, but it's not always an error. This is the main problem.

There's already some inspection for not aligned to axis parts.

<img width="1864" alt="Capture d’écran 2023-01-23 à 08 28 12" src="https://user-images.githubusercontent.com/156056/213986016-e774ebe0-1d49-411e-868e-53576d1d841c.png">
